packages/painperdu/src/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface PathItem {
```

Comment:
Utilise un `type` plutôt l'interface c'est mieux quand tu veux `extends` même si c'est relativement pareil question perf je crois que type est conseillé

Reply:
Alors de base c'est ce que j'avais mais lorsque j'utilise un `type` j'ai une erreur eslint : 
Use an `interface` instead of a `type`.eslint[@typescript-eslint/consistent-type-definitions](https://typescript-eslint.io/rules/consistent-type-definitions)

Reply:
OK ok mais plutôt utiliser les types alors tu peux configurer le linter:

https://typescript-eslint.io/rules/consistent-type-definitions/#options
https://www.youtube.com/watch?v=zM9UPcIyyhQ

PS: D'ailleurs ce mec je te conseille de le follow en TS il est TROP FORT

Reply:
L'ajout de la rule au file eslintrc ne fonctionne pas, solution envisagée :
[86b339a](https://github.com/luctst/painperdu/pull/12/commits/86b339a004635c1e47199daf4c4f56e10dc7ad29)